src/website/views/components/dropdown.pug

```diff
@@ -0,0 +1,734 @@
+            <div class="a-icon">
+              <svg>
+                <use xlink:href="#icon-logo-centurylink"></use>
+              </svg>
+            </div>
+            Element 1
+          </label>
+        </div>
+        <div class="a-formGroup m-dropdown__menu-item">
+          <input class="a-input" type="checkbox" id="checkbox2" disabled>
+          <label for="checkbox2"></label>
+          <label class="-text" for="checkbox2">
+            <div class="a-icon">
+              <svg>
+                <use xlink:href="#icon-logo-centurylink"></use>
+              </svg>
+            </div>
+            Element 2
+          </label>
+        </div>
+        <div class="a-formGroup m-dropdown__menu-item">
+          <input class="a-input" type="checkbox" id="checkbox3" checked>
+          <label for="checkbox3"></label>
+          <label class="-text" for="checkbox3">
+            <div class="a-icon">
+              <svg>
+                <use xlink:href="#icon-logo-centurylink"></use>
+              </svg>
+            </div>
+            Element 3
+          </label>
+        </div>
+        <div class="a-formGroup m-dropdown__menu-item">
+          <input class="a-input" type="checkbox" id="checkbox4">
+          <label for="checkbox4"></label>
+          <label class="-text" for="checkbox4">
+            <div class="a-icon">
+              <svg>
+                <use xlink:href="#icon-logo-centurylink"></use>
+              </svg>
+            </div>
+            Element 4
+          </label>
+        </div>
+      </div>
+
+h4 Others
+p.-text
+  span.a-badge.-brand.-mr1
+    span Pro tip
+  | You can add your content as a menu item using the class <code>m-dropdown__menu-item</code>. It sets the height, padding,
+  | and alignment to fit other menu items rendering.
+
+ul#a-tabs--others.a-tabs.-ml3
+  li.-active
+    a(href='#a-tabs-panel--others-text') Text
+  li
+    a(href='#a-tabs-panel--others-radio') Radio buttons
+  li
+    a(href='#a-tabs-panel--others-input') Text input
+  li
+    a(href='#a-tabs-panel--others-range') Range input
+  li
+    a(href='#a-tabs-panel--others-toggle') Toggle
+
+.example.-mb3
+  #a-tabs-panel--others-text.a-tabs-panel.-active
+    .-p3
+      p.-text It is possible to include text, without link, using a span tag.
+      .a-grid
+        .a-col.-w7.-sm--w4.-md--w5.-lg--w3
+          .m-dropdown__menu.-active
+            span.m-dropdown__menu-item Text 1
+            span.m-dropdown__menu-item Text 2
+            span.m-dropdown__menu-item Text 3
+            span.m-dropdown__menu-item Text 4
+    :code(lang="html")
+      <div class="m-dropdown__menu">
+        <span class="m-dropdown__menu-item">Text 1</span>
+        <span class="m-dropdown__menu-item">Text 2</span>
+        <span class="m-dropdown__menu-item">Text 3</span>
+        <span class="m-dropdown__menu-item">Text 4</span>
+      </div>
+  #a-tabs-panel--others-radio.a-tabs-panel
+    .-p3.a-grid
```

Comment:
No need for a-grid in a lot of these examples. As mentioned in previous comment, it's breaking the layout